### ISSUE TITLE
Zipped Invoices Feature

### ISSUE THREAD

**Issue by sasoriza** [firegento/firegento-pdf#371]
I am trying to include the zipped invoice feature (https://github.com/firegento/firegento-pdf/pull/244/files) into the latest development branch and although I am seeing the setting in the invoice configuration and have set this to yes, my invoices are still combined into one pdf file.

**Comment by Schrank** [comment]
I guess serveInvoicesAsZip is not called. But unfortunately I don't have time to check it :-(

**Comment by sasoriza** [comment]
Is the separate extension to provide this feature still on the cards? I think it would be very useful especially considering german tax regulations.

**Comment by Schrank** [comment]
I still have it on my mind, but don't see it implemented in my free time. Too much other stuff with higher priorities...

**Comment by sasoriza** [comment]
Would it be possible for you to realize this within a paid project?

**Comment by Schrank** [comment]
Yes, feel free to send me an email to github@fabian-blechschmidt.de but I'm on holidays from todayish until 3rd of May
